app/controllers/roster_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RosterController < ApplicationController
+  before_action :require_signed_in
+  respond_to :html, :json
+  def update_roster
```

Comment:
Where is the code that actually updates the `User` record?

Reply:
It will go in the enrollment_controller.rb file.